lib/wizard.js

```diff
@@ -60,23 +70,20 @@ const Wizard = (steps, fields, settings) => {
 
     options.i18n = settings.i18n;
 
-    const SuperClass = options.controller || settings.controller;
-
     if (options.controller) {
       deprecate(
         'hof-form-wizard: Passing a custom step controller is deprecated',
         'Instead give one or more behaviours which will be mixed in to the base controller'
       );
     }
 
-    const behaviours = options.behaviours ? _.castArray(options.behaviours) : [];
-
-    const controller = new (behaviours.length ?
-      createController(SuperClass, behaviours) :
-      SuperClass)(options);
+    const SuperClass = options.controller || settings.controller;
+    const Controller = getController(SuperClass, options.behaviours);
+    const controller = new Controller(options);
 
     controller.use([
       require('./middleware/check-session')(route, controller, steps, first),
+      require('./middleware/check-complete')(route, controller, steps, first),
```

Comment:
@lennym thinking about it - isn't first the actual first step? so in most cases `'/'`? If so wouldn't this result in `'/'` being marked as complete and redirecting to `'/'`